### PR TITLE
Create caller-links.json

### PR DESCRIPTION
**Pull request by ldonforth**
This file has the email address and website (if listed) of the callers in the Callers Directory at http://tinyurl.com/hnb72wv
it should start with a 
[
end with a 
]
and delete the comma after the final entry from the spreadsheet copy/paste

### CODE REVIEW

File: caller-links.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
+[
+{"zip": "35747",  "name": "Jane Ewing", "email": "contradance@charter.net" },
+{"zip": "85022",  "name": "Michael Barraclough", "email": "michael@michaelbarraclough.com", "website": "michaelbarraclough.com"},
```
Comment (jeffkaufman) [inline caller-links.json]: website links should start with `http://` or `https://` -- should I just assume `http://`?

Comment (jeffkaufman) [reply caller-links.json]: I'll go with assuming `http://`

**Comment by ldonforth** [comment]
I didn't police how they were formatting. My guess is assuming http:// is
probably better.

I did remove one entry where the person had listed "none" for their webpage.

Do you want me to go through and add that in when it isn't there? Or you've
got that on your side?

**Comment by jeffkaufman** [comment]
I've now updated https://www.trycontra.com/callers-heatmap to use this file, and to show little cards when you click on the markers

**Comment by ldonforth** [comment]
Thanks Jeff! Looks good :-)

I've added a link on my caller page, and I'll put up a social media post
about it.

**Pull request merged**